pkg/controller/container-runtime-config/helpers.go

```diff
@@ -281,6 +282,22 @@ func createCRIODropinFiles(cfg *mcfgv1.ContainerRuntimeConfig) []generatedConfig
 	return generatedConfigFileList
 }
 
+// updateSearchRegistriesConfig gets the ContainerRuntimeSearchRegistries data from the Image CRD
+// and creates a drop-in file for it at /etc/containers/registries.conf.d
+func updateSearchRegistriesConfig(searchRegs []string) []generatedConfigFile {
+	var (
+		generatedConfigFileList []generatedConfigFile
+		err                     error
+	)
+	tomlConf := sysregistriesv2.V2RegistriesConf{}
+	tomlConf.UnqualifiedSearchRegistries = searchRegs
+	generatedConfigFileList, err = addTOMLgeneratedConfigFile(generatedConfigFileList, searchRegDropInFilePath, tomlConf)
+	if err != nil {
+		glog.V(2).Infoln("error updating user changes for containerRuntimeSearchRegistries to registries.conf.d: ", err)
```

Comment:
If we fail here should we be a bit more vocal about the fact that this failed? Maybe return back up to `registriesConfigIgnition` and fail like all the other cases?

Reply:
We don't want to block the rest of the config updates because the drop-in file creation failed, that is why we just log it. The user can always try again if it doesn't work, but that should be a very rare case.

Reply:
Hmm ok, can we bump the log up to verbosity 1 and maybe have it as a warning? So its easier to find.

Reply:
Sure, bumped it up to 1, but looks like `Info` is the only available option for some reason

Reply:
Fixed now :)